int32 id

float32 yaw
float32 pitch
float32 roll

float32 robot_x
float32 robot_y
float32 ball_x
float32 ball_y

int32 x
int32 y

int32 ball_flag
